rehfeldmedical/metis_umbrella
# Add one to many relation between Clinic and Clinician

This PR is an exercise in simple one-to-many relationship between Clinic and Clinician.

### Changes
1. Updated Clinic and Clinician schemas to be related via clinic_id field.
2. Also updated related migrations.
3. Fixed and updated tests in `metis`
4. Updated clinician view to also show related clinic. Also fixed typos for clinician controller tests in `metis-web`.

### Testing
1. Run `mix ecto.migrate` to run the migration.
2. Run `mix ecto.rollback` to rollback last migration (run it multiple times to rollback previous migrations - once for each migration). Check between steps in the Postgres that the changes have indeed been reverted. Then run `mix ecto.migrate` again to have the latest structure in the DB.
3. Run `mix test` to verify that `metis` and `metis-web` tests work.
4. Run the app and navigate to [clinicians view](http://localhost:4000/api/clinicians) to verify that related clinic is also loaded.
